### PR TITLE
All Stia Side Task Dialogue

### PR DESCRIPTION
**Pull request by allister-translator**
take your time proofreading; im probably only like halfway done

### CODE REVIEW

File: Translated/Dialog/Katsuragi/ssn_015330.csv [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ssn_015330_0201#0,"""Please collect data on the Regional Mag. The one in Stia's Fringes, if you would."""
+ssn_015330_0202#0,"""Thank you very much!"""
+ssn_015330_0301#0,"""Oh, I see... Well, let me know when you have some time; I'd appreciate it."""
+ssn_015330_1001#0,"""Thanks so much for earlier! You really helped me out."""
+ssn_015330_1002#0,"""If something else comes up, I hope you don't mind me calling on you. When that time comes, I'd love your help again."""
+ssn_015330_1101#0,"""It's so good to see you! Actually, there's something I'd like to ask your help with again."""
+ssn_015330_1102#0,"""It's just... This time might be a little dangerous, so I'm not saying you have to."""
+ssn_015330_1201#0,"""Please collect data on the Recon Gigantix. All I need is for you to get close and confirm a visual."""
+ssn_015330_1202#0,"""The targets' strengths are unknown. Please take utmost care out there."""
+ssn_015330_1301#0,"""I seee... Well, if you change your mind, please let me know."""
```
Comment (HaiMisterGuy) [inline Translated/Dialog/Katsuragi/ssn_015330.csv]: 'I seee' a typo

File: Translated/Dialog/Katsuragi/ssn_015334.csv [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name_7448#0,"""Katsuragi"""
+ssn_015334_0010#0,"""You will?! Oh, this helps so much. Alright, please let me explain."""
+ssn_015334_0020#0,"""The other day, while in the Dreisan Plant, a scouting party encountered a certain group of DOLLS."""
+ssn_015334_0030#0,"""It seems that, compared to the average DOLLS, they had a noticeably different, heavy presence to them..."""
+ssn_015334_0040#0,"""They had no sooner engaged the DOLLS than they found themselves overpowered, with no choice but to retreat."""
```
Comment (HaiMisterGuy) [inline Translated/Dialog/Katsuragi/ssn_015334.csv]: than > that?

File: Translated/Dialog/Kirishima/ssn_015303.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name_7444#0,"""Kirishima"""
+ssn_015303_0010#0,"""You're willing to help? I appreciate it."""
+ssn_015303_0020#0,"""You see, just recently one of our investigators was left behind inside the power plant."""
+ssn_015303_0030#0,"""From what I hear, she fell behind as the team was fleeing from a DOLLS ambush..."""
+ssn_015303_0040#0,"""Chitose is her name; she's one of the trainees I've been looking after."""
+ssn_015303_0050#0,"""So, I worry for her safety..."""
+ssn_015303_0060#0,"""We were able to pick up her vitals, and can confirm she's uninjured. However, due to some malfunction with her Mag, we can't contact her."""
+ssn_015303_0070#0,"""Other troops are currently en route to rescue her. ...Alas, they seem to be short-staffed."""
+ssn_015303_0080#0,"""So I'd like you to enlist your help. Please see that she comes home safely."""
```
Comment (HaiMisterGuy) [inline Translated/Dialog/Kirishima/ssn_015303.csv]: I'd like to enlist your help

File: Translated/Dialog/Marimo/ssn_015340.csv [hunk 4]
```diff
@@ -0,0 +1,13 @@
+name_7443#0,"""Marimo"""
+ssn_015340_0001#0,"""You must be... one of the recent arrivals in Stia, yes?"""
+ssn_015340_0002#0,"""Have you been holding up well against the heat? Please remember to hydrate yourself before battle."""
+ssn_015340_0003#0,"""If you were to collapse before your enemy did, that would defeat the whole purpose."""
```
Comment (HaiMisterGuy) [inline Translated/Dialog/Marimo/ssn_015340.csv]: "that would defeat the whole purpose" doesn't make sense for me given the context. Is it suppose to suggest that it defeats the purpose of coming to Stia?

File: Translated/Dialog/Shirayuki/ssn_015481.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+name_7441#0,"""Shirayuki"""
+ssn_015481_0010#0,"""Even within my team, word of your feats is becoming a common topic."""
+ssn_015481_0020#0,"""Some of the troops would also like to study how you fight."""
+ssn_015481_0030#0,"""If it's alright with you, I want you to let them watch you fight enemies."""
```
Comment (HaiMisterGuy) [inline Translated/Dialog/Shirayuki/ssn_015481.csv]: Repetition of 'you', may read better saying

> I'd like them watch you fight enemies

File: Translated/Dialog/Souya/ssn_015310.csv [hunk 6]
```diff
@@ -0,0 +1,11 @@
+name_7446#0,"""Souya"""
+ssn_015310_0001#0,"""In a protracted war scenario, supply replenishment, as well as management of said, is more important than anything else."""
```
Comment (HaiMisterGuy) [inline Translated/Dialog/Souya/ssn_015310.csv]: May read better saying

> supplying replenishments, as well as management of so

If you rather keep the 'said' wording, then perhaps

> as well as management of said items

File: Translated/Dialog/Katsuragi/ssn_015330.csv (hunk 1 above)
Comment (allister-translator) [reply Translated/Dialog/Katsuragi/ssn_015330.csv]: fixt

File: Translated/Dialog/Katsuragi/ssn_015334.csv (hunk 2 above)
Comment (allister-translator) [reply Translated/Dialog/Katsuragi/ssn_015334.csv]: they had no sooner (done this) than (that) 
it checks out grammatically, but i could probably reword it. 
i do see a misspelled Dreisen Plant tho 👀

File: Translated/Dialog/Hatsukari/ssn_015490.csv [hunk 7]
```diff
@@ -0,0 +1,12 @@
+name_7440#0,"""Hatsukari"""
+ssn_015490_0001#0,"""Welcome to Stia!"""
+ssn_015490_0002#0,"""The call from Central was rather sudden. Sorry for the lack of a more official welcome!"""
+ssn_015490_0003#0,"""I can't speak for everyone else, but I'm glad to have ya with us!"""
+ssn_015490_0101#0,"""Yo! The name's Hatsukari. I'm the Commander of the 2nd Combat Platoon here in Stia."""
+ssn_015490_0102#0,"""For someone who ain't used to this place, but you do good work. Honestly, it'll help me out as well."""
```
Comment (HaiMisterGuy) [inline Translated/Dialog/Hatsukari/ssn_015490.csv]: -but

File: Translated/Dialog/Shioji/ssn_015510.csv [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+ssn_015510_1201#0,"""\""Inferium\"" can be gathered in Outer Mediola: Sector 1."""
+ssn_015510_1202#0,"""There's rumours that you can only find it during certain weather."""
+ssn_015510_1203#0,"""I know it's a lot to ask, but since it's you, I have faith that it'll work out."""
+ssn_015510_1301#0,"""Oh, alright... Come back if you change your mind!"""
+ssn_015510_1401#0,"""Oh, you're back. Let me start the data scan right away."""
+ssn_015510_1402#0,"""...Hum. So this is \""Inferium\"". This Ore is even more impressive than rumoured!"""
+ssn_015510_1403#0,"""It's strong, and yet it'd be easy to process."""
+ssn_015510_1404#0,"""...Ahh, just looking at it, all sorts of possibilities come to mind."""
+ssn_015510_1405#0,"""We could make some incredible things with this, there's no doubt about that."""
+ssn_015510_1406#0,"""It's been a while since the researcher in me was this fired up! I look forward to the coming days!"""
```
Comment (DabirSA) [inline Translated/Dialog/Shioji/ssn_015510.csv]: This last bit could do with rewording.

Comment (allister-translator) [reply Translated/Dialog/Shioji/ssn_015510.csv]: get ur head out of the gutter

Comment (DabirSA) [reply Translated/Dialog/Shioji/ssn_015510.csv]: Didn't even occur to me. I just think it reads a bit unnatural

**Pull request merged**